cocotb/share/makefiles/Makefile.inc

```diff
@@ -55,7 +55,10 @@ endif
 export OS
 
 # this ensures we use the same python as the one cocotb was installed into
-PYTHON_BIN ?= $(shell cocotb-config --python-bin)
+# realpath for Windows compatibility
```

Comment:
What exactly is the effect of `realpath` on windows?

Reply:
`cocotb-config --python-bin` will return `C:\User\` and `realpath` will do `/c/User` same above in the file (would need to escape `\` in cocotb-config maybe).

Reply:
Ah, so this is `cygpath -u`, to convert windows path to unix paths.

```suggestion
# realpath for to convert windows path to unix paths, like cygpath -u
```